applications/nrf5340_audio/src/bluetooth/bt_management/dfu/Kconfig

```diff
@@ -14,9 +14,6 @@ menuconfig AUDIO_BT_MGMT_DFU
 	select ZCBOR
 	select CRC
 	select MCUMGR_TRANSPORT_BT
-	select MCUMGR_GRP_ZBASIC
-	select MCUMGR_GRP_ZBASIC_STORAGE_ERASE
-	select IMG_ERASE_PROGRESSIVELY
```

Comment:
would probably want progressively as this means it doesn't have to sit waiting for the whole slot to be erased before starting upload (which can cause a timeout on some not so good applications)